lib/jest.config.js

```diff
@@ -12,11 +12,11 @@ module.exports = {
     "<rootDir>/src"
   ],
   "transform": {
-    "^.+\\.tsx?$": "ts-jest",
-    "^.+\\.jsx?$": "babel-jest"
+    "^.+\\.(t|j)sx?$": ["ts-jest", { tsconfig: '<rootDir>/tsconfig.test.json' }]
```

Comment:
I figure I'd spend a bit of time reading into what the `jest.config.js` does exactly, and I'm still a bit unsure about `transform` here. From what I gather, it helps convert JSX or Typescript we may be using into JS for jest to be able to run, and that the syntax is `{string, pathToTransformer | [pathToTransformer, object] }` - I'm just a bit unsure here as to why the `tsconfig: ...` object was added here? Just for clarity, thanks!

Reply:
>The config object with tsconfig is to tell typescript to use tsconfig.test.json instead of the default tsconfig.json.

Thanks for clarifying, that clears everything up!